designsafe/sitemaps.py

```diff
@@ -184,3 +201,14 @@ def items(self):
 
     def location(self, item):
         return item
+
+
+class DesignSafeCMSSitemap(CMSSitemap):
+    priority = .7
+    changefreq = 'weekly'
+
+    def get_urls(self, site=None, **kwargs):
+        site = Site(domain='www.designsafe-ci.org')
+        return super(CMSSitemap, self).get_urls(site=site, **kwargs)
```

Comment:
didn't catch this earlier, but I think this should be `super(DesignSafeCMSSitemap, self)`

Reply:
ah, yep good catch. @fnets can you take a look?

Reply:
Yes, one sec.

Reply:
Done. Had to open a separate PR, since this is already merged.  
https://github.com/DesignSafe-CI/portal/pull/581